src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -459,6 +506,80 @@ export default {
 			pointer-events: none;
 		}
 	}
+
+	.quote-card {
+		@include featured-text();
+
+		margin: 0 auto rem-calc(25) auto;
+		text-align: center;
+		border-radius: 1rem;
+		box-shadow: 0 0 1.2rem 1rem rgb(153, 153, 153, 0.1);
+		position: relative;
+		overflow: hidden;
+
+		@include breakpoint(large) {
+			@include medium-text();
+		}
+
+		&__lender-img {
+			margin: rem-calc(30) auto rem-calc(30) auto;
+			width: 5.5rem;
+			border-radius: 50%;
+
+			@include breakpoint(large) {
+				width: 4rem;
+			}
+		}
+
+		&__quote {
+			line-height: 1.3;
+			margin-bottom: rem-calc(30);
+		}
+
+		&__attribution {
+			line-height: 1.3;
+			font-weight: bold;
+			margin-bottom: 0;
+			z-index: 100;
+			position: relative;
+		}
+
+		&__title {
+			z-index: 100;
+			position: relative;
+			margin-bottom: rem-calc(30);
+		}
+
+		&__flourish {
+			position: absolute;
+			z-index: 0;
+		}
+	}
+
+	// Couldn't get BEM working in here
+	.quote-card:nth-child(even) {
```

Comment:
Yeah, this is a sass thing I'm afraid. What you've got is good.

In sass the` &` really just means append that to the parent selector in SASS. It makes it easy to write BEM, but it's not really BEM specific.
```
.hi {
  &-there {
```

would output `.hi-there` 

```
.hi:nth-child(even) {
  &-there {
```
would try to output `.hi:nth-child(even)-there` but fail because it's an invalid selector.


So in your case you could do
```
.quote-card:nth-child(even) {
  .quote-card {
    &__flourish { ...
```

But that seems excessive. What you've got is readable to me.

Reply:
Thanks for this explanation @ryan-ludwig